### DISCUSSION TITLE
4.2 release date?

### DISCUSSION THREAD

**Discussion by rjcampion3** [nextgenhealthcare/connect#5557] [General]
I didn't see anything on the roadmap channel. Is there a release date for version 4.2? 

Thanks

**Comment by pacmano1** [comment]
From Travis, the PM on Slack.

![image](https://user-images.githubusercontent.com/44065187/204915981-b905e8d9-1723-4392-8311-b178e1f8534f.png)

**Reply by rjcampion3** [reply]
Thank you!
